src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -230,6 +230,14 @@
   <data name="Log_InstalledSummary" xml:space="preserve">
     <value>Installed:</value>
   </data>
+  <data name="Log_AllProjectsUpToDate" xml:space="preserve">
+    <value>All projects are up-to-date</value>
```

Comment:
`All projects are up to date.`

Note the added period and the removal of hyphens.